plugins/woocommerce-admin/client/marketplace/components/product-card/product-card-footer.tsx

```diff
@@ -64,8 +64,17 @@ function ProductCardFooter( props: { product: Product } ) {
 		return true;
 	}
 
-	// We hardcode this for now while we only display prices in USD.
-	const currencySymbol = '$';
+	const currencySymbols: { [ key: string ]: string } = {
+		USD: '$',
+		AUD: 'A$',
+		CAD: 'C$',
+		EUR: '€',
+		GBP: '£',
+	};
+
+	const getCurrencySymbol = ( currencyCode: string ) => {
+		return currencySymbols[ currencyCode ] || '';
+	};
```

Comment:
I've got a two-part i18n suggestion, because _not all currencies put the currency symbol before the value_:

1. Instead of just returning the currency symbol, return a format string into which the currency can be injected -

```suggestion
	const currencyFormats: { [ key: string ]: string } = {
		USD: '$%s',
		AUD: 'A$%s',
		CAD: 'C$%s',
		EUR: '€%s',
		GBP: '£%s',
	};

	const getCurrencyFormat = ( currencyCode: string ) => {
		return currencyFormats[ currencyCode ] || '%s';
	};
```

(note I'm using `%s` in case we start sending fractional prices like `29.99` and because we can't necessarily assume the number of decimal places to which a currency is fractioned!)